### PR TITLE
Moved raw order for tag filtering to correct place

### PR DESCRIPTION
**Pull request by kirrg001**
refs #10105

- ordering !== filtering

See https://github.com/TryGhost/Ghost/issues/10105#issuecomment-438392778!

### CODE REVIEW

File: core/server/models/post.js [hunk 1]
```diff
@@ -530,14 +530,21 @@ Post = ghostBookshelf.Model.extend({
...
+        let order = '' +
             'CASE WHEN posts.status = \'scheduled\' THEN 1 ' +
             'WHEN posts.status = \'draft\' THEN 2 ' +
             'ELSE 3 END ASC,' +
             'CASE WHEN posts.status != \'draft\' THEN posts.published_at END DESC,' +
             'posts.updated_at DESC,' +
             'posts.id DESC';
+
+        // CASE: if the filter contains an `IN` operator, we should return the posts first, which match both tags
+        if (options.filter && options.filter.match(/(tags|tag):\s?\[.*\]/)) {
```
Comment (gargol) [inline core/server/models/post.js]: `\s` -> TIL we support white space in filters :sweat_smile:

Comment (kirrg001) [reply core/server/models/post.js]: i saw your test case here: https://github.com/TryGhost/Ghost/pull/10166/files#diff-5a0b10849d1a90717049d92c728caaa5R49

maybe we should remove this whitespace for now 🤔 
i am not 100% sure if it works, have not tested it on a theme

Comment (gargol) [reply core/server/models/post.js]: nql-lang picks it up, but testing in real-world scenario would def make things clear

Comment (kirrg001) [reply core/server/models/post.js]: i think for now: doesn't matter much because its optional anyway in the regex?

Comment (gargol) [reply core/server/models/post.js]: agreed. let's merge this, looks good

**Pull request merged**